ngraph/test/files/paddlepaddle/gen_scripts/generate_interpolate.py

```diff
@@ -127,7 +127,7 @@ def nearest_upsample_tensor_size():
             interp = interpolate(node_x, size=node_sizes, scale_factor=None,
                                  mode='nearest', align_corners=test['align_corners'], align_mode=test['align_mode'],
                                  data_format='NCHW', name=test['name'])
-            out = pdpd.static.nn.batch_norm(interp, use_global_stats=True, epsilon=0)
+            out = pdpd.static.nn.batch_norm(interp, use_global_stats=True)
```

Comment:
i'd suggest to return 'epsilon=0' here and/or in other places. This is a good 'edge' test where we ensure that OV supports 'epsilon' with the same range as PDPD.